R/release.r

```diff
@@ -60,6 +62,14 @@ release <- function(pkg = ".", check = TRUE, args = NULL) {
       return(invisible())
   }
 
+  if (length(spelling)) {
```

Comment:
Can you please move this to `check-devtools.r`, following the template in `release_checks()`?